fecgov/fec-cms
# [WIP] Customize record, tips, and release feed order

## Summary

- Resolves #1694 
_This updates record articles, tips, and releases to all use DateTimeField so that a time can be specified and the order can be based on that instead of just date._

- Getting an error on the updates page "datetime.datetime to datetime.date". Need to see how to convert that
- It would be good to check if this will effect the date fields for these articles, because we are introducing a time element that was not previously there.

## Impacted areas of the application
List general components of the application that this PR will affect:

- Homepage - News and announcements section
- Service landing page - Tips feed
- Updates page

## How to test
1. `./manage.py makemigrations`
2. `./manage.py migrate`
3. Change the date field of record articles, tips, and releases to see if they shift in the homepage feed and
